srcpkgs/metamath/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'metamath'
+pkgname=metamath
+version=0.179
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake"
+short_desc="Constructs mathematics from scratch, starting from ZFC"
+maintainer="Zhengqun Koo <root@zhengqunkoo.com>"
+license="GPL-2.0-or-later"
+homepage="http://us.metamath.org/mpeuni/mmset.html"
+distfiles="https://github.com/metamath/metamath-exe/archive/master.tar.gz"
```

Comment:
We need constant address, best if tagged release.

Reply:
Found link to untagged release on primary mirror, amended commit.

Reply:
This is not constant either, see earlier #15721

Reply:
Right, I didn't search before sending a pull request, sorry for the duplication.